main.development.js

```diff
@@ -6,7 +6,7 @@ let menu
 let template
 let mainWindow = null
 
-const feedUrl = `https://felony-app-update.herokuapp.com/update/${process.platform}_${process.arch}/${version}`
+const feedUrl = `https://felony-app-update.herokuapp.com/update/${ process.platform }_${ process.arch }/${ version }`
```

Comment:
Sure, do you want me to update the eslintrc or is there another PR doing that already?